scout/eia_file.py

```diff
@@ -35,7 +36,13 @@ class EIAFiles(object):
         -----------------------------
         r_lgt: Residential lighting technology data
     """
-    def __init__(self, input_dir='data', output_dir=None):
+    def __init__(self, input_dir=None, output_dir=None):
+        if not input_dir:
+            input_dir = Path(__file__).resolve().parents[1] / 'inputs'
+            output_dir = Path(__file__).resolve().parents[1] / 'inputs'
+        else:
+            input_dir = Path(input_dir)
+            output_dir = Path(output_dir)
```

Comment:
Would be better to do 

```
        if not input_dir:
            input_dir = Path(__file__).resolve().parents[1] / 'inputs'
        if not output_dir:
            output_dir = input_dir
```

Then just make sure a `Path` is passed for input_dir and output_dir (can add type hints too)